WebKitGtk/BlazorWebView.cs

```diff
@@ -125,14 +121,17 @@ void HandleUriScheme(URISchemeRequest request)
 		{
 			using var ms = new MemoryStream();
 			content.CopyTo(ms);
-			var streamPtr = MemoryInputStream.NewFromData(ref ms.GetBuffer()[0], (uint)ms.Length, _ => { });
-			var inputStream = new InputStream(streamPtr, false);
-			request.Finish(inputStream, ms.Length, headers["Content-Type"]);
+			byte[] data = ms.ToArray();
```

Comment:
Hello @loribao 

- `AsSpan(0, (int)memoryStream.Length)` provides a zero-copy slice of the buffer  
- `Span<T>` is a modern .NET construct that enables high-performance, allocation-free operations  
- This approach avoids creating intermediate objects while maintaining memory safety  

The key difference:  
- `ToArray()` creates a copy of the data in memory  
- `GetBuffer() + AsSpan()` provides direct access to the underlying buffer without additional allocations  

For large data streams, this can result in significant performance improvements and reduced memory pressure.  

You can find the optimized version here:  
https://github.com/czirok/apps/blob/f237a9b8874eac208b0503f237f2cf2218c070df/src/WebKit.BlazorWebView.GirCore/src/WebKitWebViewManager.cs#L123